frontends/web/src/components/confirm/Confirm.tsx

```diff
@@ -42,27 +52,24 @@ export default class Confirm extends Component {
         this.setState({
             active: false,
             message: '',
-        }, () => {
-            this.callback = null;
         });
     }
 
-    render({}, {
+    public render({}, {
         message,
-        callback,
         active,
     }) {
-        if (!active) return null;
```

Comment:
`if (!active) return null;` should also work with TypeScript, so changing it to `return active ? (` should not have been necessary (but I don't mind, I just wanted to make this clear).